ProtocolHandler.pm

```diff
@@ -348,6 +348,18 @@ sub getMetadataFor {
 			}
 		}
 	}
+	
+	if ( $prefs->get('parentalWarning') && $meta->{parental_warning} ) {
+		$meta->{title} .= ' [E]';
+	}
+
+	if ( $prefs->get('discNumber') && $meta->{media_count} > 1 && $prefs->get('trackNumber') ) {
+		$meta->{title} = "Disc $meta->{media_number}/$meta->{track_number}: $meta->{title}";
+	} elsif ( $prefs->get('discNumber') && $meta->{media_count} > 1 ) {
+		$meta->{title} = "Disc $meta->{media_number}: $meta->{title}";
+	} elsif ( $prefs->get('trackNumber') ) {
+		$meta->{title} = "$meta->{track_number}: $meta->{title}";
+	}
```

Comment:
Hmm... this could have a negative impact on MAI. "Disc 1/12: My Song" would definitely not be the same as "My Song" when needed to be looked up by title.